src/lib.rs

```diff
@@ -1902,7 +1902,13 @@ impl Build {
 
         // Target flags
         match cmd.family {
-            ToolFamily::Clang { .. } => {
+            ToolFamily::Clang { zig_cc } => {
+                let target = if zig_cc {
+                    Cow::Owned(target.replace("-unknown-", "-"))
+                } else {
+                    Cow::Borrowed(target)
+                };
+
```

Comment:
Could do a

```suggestion
            let target = target.as_ref();

```

To avoid changing every if that uses target